content/en/news/releases/1.3.x/announcing-1.3.6/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.3.6
+description: Istio 1.3.5 release announcement.
```

Comment:
```suggestion
description: Istio 1.3.6 release announcement.
```

Reply:
```suggestion
description: Istio 1.3.6 patch release.
```

Please look at the current 1.3.5 release announcement for a current example of what to include in the front-matter. Looks like you copied an older version of the markdown file.

Reply:
Thanks. Updated to the new style.